includes/wp-admin/import/class-starter-kit.php

```diff
@@ -242,19 +242,23 @@ public static function follow() {
 
 		$items = self::$starter_kit['items'] ?? self::$starter_kit['orderedItems'] ?? array();
 
-		foreach ( $items as $item ) {
-			if ( ! is_actor( $item ) ) {
+		foreach ( $items as $actor_id ) {
+			if ( ! \is_string( $actor_id ) ) {
+				$actor_id = object_to_uri( $actor_id );
+			}
```

Comment:
```suggestion
			$actor_id = object_to_uri( $actor_id );
```

This check is already done by `object_to_uri`.

Reply:
Indeed! Fixed in [9fcb478](https://github.com/Automattic/wordpress-activitypub/pull/2039/commits/9fcb4781547db6facdbae1b4975fb2e97447448e).